x/group/keeper/keeper_test.go

```diff
@@ -2443,6 +2446,138 @@ func (s *TestSuite) TestExecProposal() {
 	}
 }
 
+func (s *TestSuite) TestVPEndProposals() {
```

Comment:
this test could actually be more appropriate to add in some `x/group/abci_test.go` since you're testing `EndBlocker` directly and not the `keeper.UpdateTallyOfVPEndProposals` method